### PR TITLE
Change to backup to root UFS by default

### PR DESCRIPTION
**Pull request by LuQQiu**
Backup to root UFS by default and modify the docs

### CODE REVIEW

**Comment by ns1123** [comment]
@LuQQiu what happens if the user doesn't have permissions for root UFS default location? For example, if my Root UFS for Alluxio is 'hdfs://host:port/dir/alluxio' and Alluxio doesn't have write permissions for 'hdfs://host:port/alluxio_backups/'?

**Comment by LuQQiu** [comment]
they can change the default backup location or directly pass in the folder path that they have permission

**Comment by LuQQiu** [comment]
@ns1123 @ggezer PTAL, thanks!

**Comment by ns1123** [comment]
LGTM

**Comment by LuQQiu** [comment]
@apc999 PTAL, thanks

**Comment by ZacBlanco** [comment]
How would I backup to my local filesystem after this change takes effect (this affects all current EMR deployments)

**Comment by LuQQiu** [comment]
@ZacBlanco Use `./bin/alluxio fsadmin backup /opt/alluxio/backups/ --local [dir]` to backup to local

**Comment by LuQQiu** [comment]
Tests in error: 
alluxio.client.fs.FreeAndDeleteIntegrationTest.deleteDir(alluxio.client.fs.FreeAndDeleteIntegrationTest)
  Run 1: FreeAndDeleteIntegrationTest.deleteDir:146 » TestTimedOut test timed out after...
  Run 2: FreeAndDeleteIntegrationTest.deleteDir:145 » TestTimedOut test timed out after...

**Comment by LuQQiu** [comment]
jenkins, test this please

**Comment by LuQQiu** [comment]
@apc999 PTAL, I resolve the conflicts coming from your PR https://github.com/Alluxio/alluxio/pull/9808
Thanks

File: core/common/src/main/java/alluxio/conf/PropertyKey.java [hunk 1]
```diff
@@ -1207,7 +1207,7 @@ public String toString() {
           .build();
   public static final PropertyKey MASTER_BACKUP_DIRECTORY =
       new Builder(Name.MASTER_BACKUP_DIRECTORY)
-          .setDefaultValue(String.format("${%s}/alluxio_backups", Name.WORK_DIR))
+          .setDefaultValue("/alluxio_backups")
```
Comment (apc999) [inline core/common/src/main/java/alluxio/conf/PropertyKey.java]: should this be `String.format("${%s}/alluxio_backups", Property.MASTER_MOUNT_TABLE_ROOT_UFS. getDefaultValue()`?

**Comment by apc999** [comment]
Left one comment and  adding the TODO not using the factory method to get the local fs adaptor. otherwise, LGTM

File: core/common/src/main/java/alluxio/conf/PropertyKey.java (hunk 1 above)
Comment (LuQQiu) [reply core/common/src/main/java/alluxio/conf/PropertyKey.java]: We want to back up to the root ufs.
Like  root ufs maybe is `hdfs://host:port/alluxio/`
The backup will backup to `hdfs://host:port/alluxio_backups/`

Comment (LuQQiu) [reply core/common/src/main/java/alluxio/conf/PropertyKey.java]: The backup folder should not under alluxio userspace ideally. We don't want users to use alluxio API to manage their backups

Comment (apc999) [reply core/common/src/main/java/alluxio/conf/PropertyKey.java]: ack

**Comment by apc999** [comment]
alluxio-bot, merge this please

**Pull request merged**